docs/amd_sev_snp.md

```diff
@@ -0,0 +1,38 @@
+# AMD SEV-SNP
+
+### WARNING
+This feature is only supported on MSHV and not KVM for the time being.
```

Comment:
You can just say "This feature is only currently supported on MSHV"

Reply:
Done